cmd/cosign/cli/options/attach.go

```diff
@@ -58,7 +58,7 @@ func (o *AttachSignatureOptions) AddFlags(cmd *cobra.Command) {
 			"signing certificate and end with the root certificate. Included in the OCI Signature")
 	cmd.Flags().StringVar(&o.TimeStampedSig, "tsr", "",
 		"path to the Time Stamped Signature Response from RFC3161 compliant TSA")
-	cmd.Flags().StringVar(&o.RekorBundle, "rekor-response", "",
+	cmd.Flags().StringVar(&o.RekorResponse, "rekor-response", "",
 		"path to the rekor bundle")
```

Comment:
Can you update the description to note this can either be the output from `cosign sign --bundle` or a Rekor response formatted as a RekorBundle struct?